internal/oauth/oauth.go

```diff
@@ -47,6 +47,12 @@ var defaultTransport = &http.Transport{
 	ExpectContinueTimeout: expectContinueTimeout,
 }
 
+var validHostnames = map[string]bool{
+	config.NativeHostName:    true,
+	config.ContainerHostName: true,
+	config.ActionsHostName:   true,
```

Comment:
Action is quite generic and if someone does not have context, it may not be obvious that we are taking about github action. 
```suggestion
	config.GitHubActionsHostName:   true,
```
Another name could be CIPipeline